internal/core/src/common/FieldMeta.h

```diff
@@ -139,13 +155,26 @@ class FieldMeta {
         return type_ == DataType::VECTOR_BINARY || type_ == DataType::VECTOR_FLOAT;
     }
 
+    bool
+    is_string() const {
```

Comment:
this API seems duplicate with datatype_is_string()

Reply:
datatype_is_string() is a util function, not a member function of fieldMeta